python/tests/src/tests/kat/t_active_health_check.py

```diff
@@ -122,7 +128,7 @@ def check(self):
                 valid += 1
             elif self.results[i].status == 500:
                 errors += 1
-        msg = "Errors: {}, Valid: {}".format(errors, valid)
+        # msg = "Errors: {}, Valid: {}".format(errors, valid)
```

Comment:
Should just delete this. Originally I think it was used as the assertion string (cf the `ActiveHealthCheckTest` function earlier in this file) but it's a terrible string for that anyway.